SUAPI-CppWrapper/Geometry.hpp

```diff
@@ -25,6 +25,12 @@
 // SOFTWARE.
 //
 
+// Remove Visual Studio min and max macros
+#ifdef WIN32
```

Comment:
I think global min/max isn't coming from VS, but from something including windows.h.

If the project define `#define NOMINMAX`, then that might fix it. to ensure it's there before any includes are done it might be best to place it in the project configuration instead of source code.